Doc/conf.py

```diff
@@ -92,19 +92,26 @@
 
 # Get LaTeX to handle Unicode correctly
 latex_elements = {
-    'inputenc': r'\usepackage[utf8x]{inputenc}',
-    'utf8extra': '',
-    'fontenc': r'\usepackage[T1,T2A]{fontenc}',
+    'fontenc': r'\usepackage[T2A,TS1,T1]{fontenc}',
 }
 
-# Additional stuff for the LaTeX preamble.
+# Additional stuff for the LaTeX preamble (inc. handling of stray Unicode chars)
 latex_elements['preamble'] = r'''
 \authoraddress{
   \sphinxstrong{Python Software Foundation}\\
   Email: \sphinxemail{docs@python.org}
 }
 \let\Verbatim=\OriginalVerbatim
 \let\endVerbatim=\endOriginalVerbatim
+\ifx\XeTeXinterchartoks\undefined % not for xelatex
+\ifx\directlua\undefined          % not for lualatex
+\ifx\kanjiskip\undefined          % not for platex
+\usepackage{newunicodechar}
+\newunicodechar{ſ}{{\fontencoding{TS1}\fontfamily{lmr}\selectfont s}}
+\newunicodechar{K}{\ensuremath{\mathrm K}}
```

Comment:
@JulienPalard in this line when one copy pastes from inside Firefox from the github web view one gets a standard ascii K in as first argument to `\newunicodechar`. But in my commit it really is U+212A (KELVIN SIGN).

Reply:
Thanks for notifying. I used curl | git apply if I remember correcly and everything is fine. Errors I still have come from translations strings on 2.7 so we should just fix them in our translation repositories.

Reply:
I got trapped myself because I copied pasted from the web view directly to `Doc/conf.py` rather than git applying my patch to another branch.